trace/setup.py

```diff
@@ -0,0 +1,44 @@
+"""A setup module for the GAPIC Stackdriver Trace API library.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+from setuptools import setup, find_packages
+import sys
+
+install_requires = [
+    'google-gax>=0.15.7, <0.16dev',
+    'googleapis-common-protos[grpc]>=1.5.2, <2.0dev',
+]
+
+setup(
+    name='google-cloud-trace',
+    version='0.15.4',
```

Comment:
All the GAPICs more or less follow the same versioning scheme right now, and 0.15.4 is where we are.

We now (as of last month) have the ability to override this on a per-API basis, so maybe we should start this at 0.1.0?